geoalchemy2/admin/dialects/mysql.py

```diff
@@ -192,3 +192,22 @@ def _MySQL_ST_GeomFromWKB(element, compiler, **kw):
 @compiles(functions.ST_GeomFromEWKB, "mysql")  # type: ignore
 def _MySQL_ST_GeomFromEWKB(element, compiler, **kw):
     return _compile_GeomFromWKB_MySql(element, compiler, **kw)
+
+@compiles(functions.ST_GeomFromText, "mariadb")  # type: ignore
```

Comment:
I think that it's possible to combine these decorators like:
```python
@compiles(functions.ST_GeomFromText, "mysql")  # type: ignore
@compiles(functions.ST_GeomFromText, "mariadb")  # type: ignore
def _MySQL_ST_GeomFromText(element, compiler, **kw):
    return _compile_GeomFromText_MySql(element, compiler, **kw)
```
Can you test it please?

Reply:
Yup